### ISSUE TITLE
Implement Black Formatter

### ISSUE THREAD

**Issue by adamgulde** [ufosc/TERMINALMONOPOLY#51]
"Black is the uncompromising Python code formatter. By using it, you agree to cede control over minutiae of hand-formatting. In return, Black gives you speed, determinism, and freedom from pycodestyle nagging about formatting. You will save time and mental energy for more important matters." from Black readme. [github here](https://github.com/psf/black)

Having a formatter will make all our functions look so much nicer. As appealing as the open-source, ununified look is, I might have a brain aneurysm if I have to look at my own code (and others'...) in the same way it is now. 

Tl;dr: black makes code pretty. we need it
